pkg/acceptance/helpers/context_client.go

```diff
@@ -33,6 +33,35 @@ func (c *ContextClient) CurrentAccount(t *testing.T) string {
 	return currentAccount
 }
 
+func (c *ContextClient) CurrentAccountName(t *testing.T) string {
```

Comment:
If you are making at least two separate calls to get current context of account name, org name, account id. I would prefer to use CurrentSessionDetails in ContextFunctions client interface (one bigger query is faster than two or more smaller ones).